index.html

```diff
@@ -9008,6 +9029,9 @@ <h2>Context Definitions</h2>
     its value MUST be an <a>absolute IRI</a>, a <a>relative IRI</a>,
     or <a>null</a>.</p>
 
+  <p class="changed">If the <a>context definition</a> has an <code>@type</code> key,
+    its value MUST be a <a>dictionary</a> with the single member <code>@context</code> with value <code>@set</code>.</p>
```

Comment:
Should be `@container`.  Probably should also be `with a value of` or `with the value`.

Reply:
Or maybe `<code>@container</code> set to <code>@set</code>` to align with the api text?